include/swift/AST/ASTContext.h

```diff
@@ -962,6 +963,15 @@ class ASTContext final {
   /// Each kind and SourceFile has its own cache for a Type.
   Type &getDefaultTypeRequestCache(SourceFile *, KnownProtocolKind);
 
+  using SILTransformCtors = ArrayRef<SILTransform *(*)(void)>;
```

Comment:
SILTransformCtors => SILTransformConstructor?

Reply:
@gottesmm Hmm, I'd prefer to keep the name plural as it's referring to an array – `SILTransformConstructors` perhaps?